html/changelogs/Kurfurst-pillbalance.yml

```diff
@@ -0,0 +1,5 @@
+author: Kurfurst
+delete-after: True
+changes:
+ - tweak: Time-release pills are now NOREACT.
+ - bugfix: Xenoarch machines can now have their panel opened and be moved about with a wrench. By extension, you can now deconstruct them and upgrade them.
```

Comment:
He commented line 5 though, he's obviously commenting on the xenoarch hotfix, not on line 4

There is a better explanation of what noreact does on the pill itself when examined anyway